### PR TITLE
(BB-559) Modify PR template to remind author if there's a need to update a con…

### PR DESCRIPTION
**Pull request by xirdneh**
…figuration-secure repo.

Configuration Pull Request
---

Make sure that the following steps are done before merging

  - [ ] A devops team member has commented with :+1:
  - [ ] are you adding any new default values that need to be overridden when this goes live?
    - [ ] Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
    - [ ] Add an entry to the CHANGELOG.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ Make sure that the following steps are done before merging
   - [ ] are you adding any new default values that need to be overridden when this goes live?
```
Comment (lgp171188) [inline .github/PULL_REQUEST_TEMPLATE.md]: @xirdneh, this already conveys what we want. Can we use it as-is?

**Comment by lgp171188** [comment]
@xirdneh, since this PR is aimed at the `master` branch, will it work on all the client-specific branches or do we have to create separate PRs for those as well?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (xirdneh) [reply .github/PULL_REQUEST_TEMPLATE.md]: I think we could just keep that one. I thought we wanted something more explicit.

**Comment by xirdneh** [comment]
@lgp171188 Yes, in fact only templates created in the default branch are used: https://help.github.com/en/articles/about-issue-and-pull-request-templates

**Review by lgp171188 — APPROVED** [review]
@xirdneh 
:+1: 
- [x] ~~I tested this~~ NA
- [x] I read through the change
- [x] ~~I checked for accessibility issues~~ NA
- [x] Includes documentation

**Pull request merged**